angular-exercise-6/src/app/app.component.html

```diff
@@ -0,0 +1,28 @@
+<h1>WELCOME TO FORM!</h1>
+<form [formGroup]="registerForm" (ngSubmit)="onSubmit()" class="form">
+  <div class="field">
+    <label class="label">username</label>
+    <input type="text" class="input" formControlName="username" [ngClass]="{'invalid': registerForm.get('username')?.invalid && registerForm.get('username')?.touched}" />
```

Comment:
Create a function that checks all these conditions and returns only a boolean to simplify reading the html code